src/EditorFeatures/CSharpTest/SimplifyInterpolation/SimplifyInterpolationTests.cs

```diff
@@ -799,6 +799,101 @@ public async Task PadLeftSimplificationIsStillOfferedOnBaseToString()
 @"class C
 {
     public override string ToString() => $""Test: {base.ToString(),10}"";
+}");
+        }
+
+        [Fact, WorkItem(42887, "https://github.com/dotnet/roslyn/issues/42887")]
+        public async Task FormatComponentSimplificationIsNotOfferedOnNonIFormattableType()
```

Comment:
trait?

Reply:
Traits have all been declared on the class header because I hate boilerplate. You okayed this at one point :D